Created a project with an empty activity

ppolly28 opened the pull request.

ppolly28 committed:
Created a project with an empty activity

65Android commented on .idea/gradle.xml:
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```
файлы из папки ./idea не нужны в репозитории, давай их удалим

ppolly28 replied:
Исправила

65Android commented on MyApplication/.idea/.name:
```diff
@@ -0,0 +1 @@
+My Application
```
файлы из папки ./idea не нужны в репозитории, давай их удалим

ppolly28 replied:
Исправила

65Android commented on MyApplication/app/build.gradle:
```diff
@@ -0,0 +1,43 @@
+    compileSdk 32
+
+    defaultConfig {
+        applicationId "com.example.myapplication"
```
поменяй applicationId, не используй com.example
например на com.ppolly.contactapp

65Android commented on MyApplication/app/src/main/AndroidManifest.xml:
```diff
@@ -0,0 +1,27 @@
+        tools:targetApi="31">
+        <activity
+            android:name=".MainActivity"
+            android:exported="true">
```
android:exported="true" этот тег точно здесь нужен?)

ppolly28 replied:
При попытке убрать эту строчку кода Android Studio ругается и в вариантах исправления ошибки просит её вернуть

65Android commented on MyApplication/app/src/main/res/values/strings.xml:
```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="app_name">My Application</string>
```
название приложение ничего о нем не говорит, давай поменяем на что то другое)
например на Contact App

65Android commented on MyApplication/settings.gradle:
```diff
@@ -0,0 +1,16 @@
+        mavenCentral()
+    }
+}
+rootProject.name = "My Application"
```
нужно поменять название

65Android commented on MyApplication/app/src/main/java/com/example/myapplication/MainActivity.kt:
```diff
@@ -0,0 +1,11 @@
+package com.example.myapplication
```
также после смены applicationId нужно будет поменять package, во всех файлах

ppolly28 committed:
Created a new project with an empty activity and a correct name

При попытке переименовать проект и поменять applicationId происходило что-то странное, поэтому я решила создать новый

ppolly28 committed:
Remove .idea

ppolly28 committed:
Remove .idea
